Stylesheets/common _css/utility.css

```diff
@@ -82,6 +82,14 @@
     align-items: center;
 }
 
+.centered-column {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
```

Comment:
Avoid giving extra space.

Reply:
Hi Tanishq
the autofill/suggestion for vscode gives those spaces.
It is present in all of my css and it won't be possible to change all of the css.
I have noted your point and further on keep a tab on it.
Thanks